WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -94,6 +95,11 @@ class PageListViewModel @Inject constructor(
     private val PageModel.isPostsPage: Boolean
         get() = remoteId == pagesViewModel.site.pageForPosts
 
+    private val showAuthorName: Boolean by lazy {
+        // show if the site is a single user site and the users in the site has the capability to edit/add pages
+        !pagesViewModel.site.isSingleUserSite && pagesViewModel.site.hasCapabilityEditOthersPages
```

Comment:
❓ Should this also include the `site.isUsingWpComRestApi` check?

Reply:
I don't think thats needed. Even for a self hosted site, the author name was being shown earlier and I don't think we should change that. I tested the changes in this PR and the trunk using a Jurassic site with multiple users

Reply:
Gotcha, I thought this was wrapping whether the filter should be shown. 🙇